components/feature/readerview/src/test/java/mozilla/components/feature/readerview/view/ReaderViewControlsBarTest.kt

```diff
@@ -185,4 +185,12 @@ class ReaderViewControlsBarTest {
 
         verify(listener).onColorSchemeChanged(ReaderViewFeature.ColorScheme.SEPIA)
     }
+
+    @Test
+    fun `tryInflate is only successfully once`() {
+        val bar = ReaderViewControlsBar(appCompatContext)
+
+        assertTrue(bar.tryInflate())
```

Comment:
nit: I don't love that the return value is not used in production. As such, I'd prefer to remove it and instead write a test that mocks `context.getService<LayoutInflater>` (or whatever) to verify the inflation occurs.

That being said, we're pressed for time and this is a trivial detail so I actually don't think it's that important to fix this.

Reply:
> That being said, we're pressed for time and this is a trivial detail so I actually don't think it's that important to fix this.

I agree, but maybe it'll be more useful if Fenix wants to "warm up" later?

(Spitballing ideas to make it feel worth it :)